aws_secretsmanager_agent/src/config.rs

```diff
@@ -620,4 +639,16 @@ mod tests {
         ))
         .unwrap();
     }
+
+    /// Tests that validate_credentials defaults to true and can be overridden
+    #[test]
+    fn test_validate_credentials() {
```

Comment:
I think the tests above already cover this right?

Reply:
You're right. I have removed that test and ensured the existing tests are properly checking for validate_credentials. Please check and let me know if it needs any further edits. Thank you!